numba_cuda/numba/cuda/runtime/nrt.py

```diff
@@ -32,6 +32,11 @@
     config.CUDA_ENABLE_NRT = ENABLE_NRT
 
 
+def get_include():
```

Comment:
Just a nit: I think a name like `get_nrt_include_path` is more specific to what this does.

Reply:
This is made to mirror similar APIs like [numpy get_include()](https://numpy.org/devdocs/reference/generated/numpy.get_include.html) or [pyarrow get_include()](https://arrow.apache.org/docs/python/generated/pyarrow.get_include.html)

Reply:
Ah I was not aware. It looks good then!